egs2/l3das22/enh1/local/l3das22_multi_channel.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+. ./path.sh
+
+
+
+if [ $# -ne 1 ]; then
+  echo "Arguments should be L3DAS wav path, see local/data.sh for example."
+  exit 1;
+fi
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# configs
+nj=16
```

Comment:
If this is a parsable argument, it is better to add a `help_message` string like in [egs2/chime4/enh1/local/data.sh#L13-L30](https://github.com/espnet/espnet/blob/master/egs2/chime4/enh1/local/data.sh#L13-L30).

Reply:
Thanks! Added in 
https://github.com/neillu23/espnet/blob/l3das22/egs2/l3das22/enh1/local/l3das22_multi_channel.sh